readme.md

```diff
@@ -9,7 +9,7 @@ For information/issues about cellxgene and its portal please refer to:
 
 ## Installation
 
-The central tool provided here is a CLI for augmenting datasets with the [cellxgene schema](cellxgene_schema_cli/cellxgene_schema/schema_definitions/2_0_0.yaml) so they can be hosted at [cellxgene's portal](https://cellxgene.cziscience.com/). 
+The central tool provided here is a CLI that validaties datasets follow the [cellxgene schema](single-cell-curation/schema/2.0.0/corpora_schema.md) so they can be hosted at [cellxgene's portal](https://cellxgene.cziscience.com/). 
```

Comment:
The primary curation tool is the `cellxgene-schema` CLI. It enables curators to perform [schema](single-cell-curation/schema/2.0.0/corpora_schema.md) validation for datasets to be hosted on the [cellxgene Data Portal](https://cellxgene.cziscience.com/).

**Editorial Note to Pablo**: Ambrose was proposing this morning to rename the cellxgene schema to "schema".